charts/kubefed/charts/controllermanager/templates/webhook.yaml

```diff
@@ -137,3 +150,49 @@ type: kubernetes.io/tls
 stringData:
   tls.crt: {{ $cert.Cert | quote }}
   tls.key: {{ $cert.Key | quote }}
+{{- else }}
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: {{ .Release.Name }}-ca-issuer
+spec:
+  selfSigned: {}
+---
+# Generate a CA Certificate used to sign certificates for the webhook
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: {{ .Release.Name}}-root-certificate
+spec:
+  secretName: {{ .Release.Name }}-root-ca
+  duration: 8760h # 1y
+  issuerRef:
+    name: {{ .Release.Name }}-ca-issuer
+  commonName: "ca.webhook.kubefed"
+  isCA: true
+---
+# Create an Issuer that uses the above generated CA certificate to issue certs
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: {{ .Release.Name }}-issuer
+spec:
+  ca:
+    secretName: {{ .Release.Name}}-root-ca
+---
+# Finally, generate a serving certificate for the webhook to use
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: {{ .Release.Name }}-certificate
+spec:
+  secretName: kubefed-admission-webhook-serving-cert
+  duration: 8760h # 1y
+  issuerRef:
+    name: {{ .Release.Name }}-issuer
+  dnsNames:
+    - kubefed-admission-webhook
+    - {{ $altName1 }}
```

Comment:
Where are these supposed to be filled from ?

Reply:
from the top of the file: `{{- $altName1 := printf "kubefed-admission-webhook.%s" .Release.Namespace }}`